tests/pytest/core/admin/test_common.py

```diff
@@ -0,0 +1,56 @@
+import pytest
+
+from django.contrib import admin
+
+from benefits.core import models
+from benefits.core.admin.common import PemDataAdmin
+
+
+@pytest.fixture
+def admin_model():
+    return PemDataAdmin(models.PemData, admin.site)
+
+
+@pytest.mark.django_db
+@pytest.mark.parametrize(
+    "user_type,expected",
+    [("staff", ["remote_url", "text_secret_name"]), ("super", None)],
+)
+def test_exclude_fields(admin_user_request, admin_model, user_type, expected):
+    request = admin_user_request(user_type)
+
+    exclude = admin_model.get_exclude(request)
+
+    if expected:
+        assert set(exclude) == set(expected)
+    else:
+        assert exclude is None
+
+
+@pytest.mark.django_db
+@pytest.mark.parametrize(
+    "user_type,expected",
+    [("staff", ["label"]), ("super", ())],
+)
+def test_readonly_fields(admin_user_request, admin_model, user_type, expected):
+    request = admin_user_request(user_type)
+
+    readonly = admin_model.get_readonly_fields(request)
+
+    assert set(readonly) == set(expected)
+
+
+@pytest.mark.django_db
+@pytest.mark.parametrize(
+    "user_type,expected",
+    [("super", True)],
```

Comment:
We should probably have at least one `False` test case here right?

Reply:
Yep, I overlooked having a `staff` `user_type` yet we need to test that case too, thanks for the reminder! It's now here https://github.com/cal-itp/benefits/blob/3b315b26d5c672323cdefe4e912525e0c5eb8eb4/tests/pytest/core/admin/test_common.py#L33